meeteval/io/stm.py

```diff
@@ -72,6 +72,16 @@ def serialize(self):
         return (f'{self.filename} {self.channel} {self.speaker_id} '
                 f'{self.begin_time} {self.end_time} {self.transcript}')
 
+    def segment_dict(self):
+        """Returns a segment dict"""
```

Comment:
I changed my mind, I think we should mention, that this is CHiME-7 style, although the type of start_time and end_time is wrong.
Otherwise, the field names are strange. `session_id` is CHiME specific, and it will likely confuse a user, that we changed most key names.